tests/integration/test_data/cached_dependencies.yaml

```diff
@@ -197,3 +197,104 @@ gomod_cached_deps:
     - "pkg:golang/rsc.io%2Fsampler@v1.3.0"
     source_purls:
     - "pkg:golang/github.com%2Fcachito-testing%2Fcachito-gomod-without-deps@DEP_VERSION"
+# Test npm with cached dependencies
+npm_cached_deps:
+  # Use local version of repos.
+  # <use_local: True> is not supported in local environment and will be skipped.
+  use_local: True
+  # test repo user
+  git_user: "Arthur Dent"
+  # test repo user email
+  git_email: "dent42@cachito.rocks"
+  # HTTPS and SSH links to main and dependency repos respectively
+  https_main_repo: https://github.com/cachito-testing/cachito-npm-with-deps.git
+  https_dep_repo: https://github.com/cachito-testing/cachito-npm-without-deps.git
+  ssh_main_repo: "git@github.com:cachito-testing/cachito-npm-with-deps.git"
+  ssh_dep_repo: "git@github.com:cachito-testing/cachito-npm-without-deps.git"
+  pkg_managers: [ "npm" ]
+  # Parts of the Cachito response to check
+  response_expectations:
+    dependencies:
+    - name: "github.com/cachito-testing/cachito-npm-without-deps"
+      replaces: null
+      type: "npm"
+      version: ""
+    packages:
```

Comment:
And the same for the packages (see the [test data](https://github.com/release-engineering/cachito/blob/master/tests/integration/test_data/npm_packages.yaml#L94):
```
"packages": [
    {
      "dependencies": [
        {
          "dev": false,
          "name": "assertion-error",
          "replaces": null,
          "type": "npm",
          "version": "1.1.0"
        },
        {
          "dev": false,
          "name": "cachito-npm-without-deps",
          "replaces": null,
          "type": "npm",
          "version": "github:cachito-testing/cachito-npm-without-deps#2f0ce1d7b1f8b35572d919428b965285a69583f6"
        },
        {
          "dev": false,
          "name": "chai",
          "replaces": null,
          "type": "npm",
          "version": "4.2.0"
        },
        {
          "dev": false,
          "name": "check-error",
          "replaces": null,
          "type": "npm",
          "version": "1.0.2"
        },
        {
          "dev": false,
          "name": "deep-eql",
          "replaces": null,
          "type": "npm",
          "version": "3.0.1"
        },
        {
          "dev": false,
          "name": "fecha",
          "replaces": null,
          "type": "npm",
          "version": "https://github.com/taylorhakes/fecha/archive/91680e4db1415fea33eac878cfd889c80a7b55c7.tar.gz"
        },
        {
          "dev": false,
          "name": "get-func-name",
          "replaces": null,
          "type": "npm",
          "version": "2.0.0"
        },
        {
          "dev": false,
          "name": "pathval",
          "replaces": null,
          "type": "npm",
          "version": "1.1.1"
        },
        {
          "dev": false,
          "name": "test-npm-dep",
          "replaces": null,
          "type": "npm",
          "version": "git+https://elinah@bitbucket.org/elinah/test-npm-dep.git#b952715c0706f30d92a7253dcdadc510a618eca5"
        },
        {
          "dev": false,
          "name": "type-detect",
          "replaces": null,
          "type": "npm",
          "version": "4.0.8"
        },
        {
          "dev": true,
          "name": "npm-test-project",
          "replaces": null,
          "type": "npm",
          "version": "git+https://gitlab.com/eakhmano-test-group/eakhmano-test-subgroup/subsubgroup/npm-test-project.git#88114c48d7d894f7e2306adf06137b096e9cdf29"
        }
      ],
      "name": "cachito-npm-with-deps",
      "type": "npm",
      "version": "1.0.0"
    }
  ],
```

Reply:
There could be a reasonable question: how to get Cachito API data for your repo? Just run Cachito locally and submit a request with your repo. See https://github.com/release-engineering/cachito#quick-start